src/bindings/python/CMakeLists.txt

```diff
@@ -115,6 +120,9 @@ set(INIT_FILES_RUNTIME "${OpenVINOPython_SOURCE_DIR}/src/openvino/__init__.py"
 ov_check_init_files_alignment("${INIT_FILES_RUNTIME}")
 
 ov_option(ENABLE_PYTHON "Enables OpenVINO Python API build" ${ENABLE_PYTHON_DEFAULT})
+message(BBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB)
+message("${ENABLE_PYTHON}")
+message("${ENABLE_PYTHON_DEFAULT}")
 
```

Comment:
what issue are you trying to debug?

Reply:
Conda build https://github.com/conda-forge/openvino-feedstock/pull/100